### PR TITLE
Human misc augments

### PR DESCRIPTION
**Pull request by ghost**
Hey kids, ever think to yourself "Gee, I really love hacking and welding, but lugging multitools and welding goggles around is a real pain!"

Well now you can augment yourself with cool tools from robotics to make yourself into more machine than man!

**Hacking Augment:**
Removes the need for a multitool when hacking.
**Welding Augment:**
Removes the need for a welding visor when welding. A toggle-implant.
**Flashlight Augment:**
Lights you up like a bulb on demand.

You can ghetto surgery some welding goggles or a multitool into your head if you want to augment yourself, but I don't recommend it because you could end up blind or brain dead. There's always someone stupid enough to try it of course!

### CODE REVIEW

**Comment by Deantwo** [comment]
Wait what? That sound really painful.

At least surgery will have an new interesting use? Or what?

**Comment by GunHog** [comment]
This should be an augmentation, not simply 'shove item into your head to suddenly become Adam Jensen'.

**Comment by Deantwo** [comment]
Yeah, like something R&D makes or robotics.
You can already get robotic arms and legs right?
Maybe just upgraded version of one of those for the hacking thing? Makes more sense than having the multitool in your head.

**Comment by paprka** [comment]
Making these augmentations or robotics-specialized would cut out the ability to do them via ghetto surgery, but that would also make robotics more useful. I might rework them to do that instead, it's just as easy. Though I'd need some custom sprites for the welding/hacking augments themselves.

**Comment by Jalleo** [comment]
Could the goggles be a specific eye surgery version (I cant tell if it is already due to not knowing surgery code at all)

**Comment by Jalleo** [comment]
Nevermind my previous comment if you still see it if you do those planned changes you get :+1: until then :bowtie:

**Comment by AlexCrimson** [comment]
Is there any outside indication that they have these things implanted?

**Comment by Deantwo** [comment]
Still think the multitool should be in the arm rather than the head.

**Comment by paprka** [comment]
>not hooking wires up to your skull

you don't even cyberpunk

anyway OP updated

**Comment by paprka** [comment]
Ok they're augments now. Robotics is relevant more now I guess?

**Comment by GunHog** [comment]
Delicious :+1:

**Comment by Stickymayhem** [comment]
Is this as simple as being able to interact with the hacking menus with nothing in hand?

**Comment by PKPenguin321** [comment]
So just a really fast rundown, you can use science-made professional augments for the official augmentation or use a regular multitool/welding goggles for ghetto surgery?

Two things if that's the case. One, I don't think there's mention for a ghetto flashlight augment, while the other two have ghetto replacements.
And two, welding goggles are (I'm pretty sure) only available from roundstart in robotics and Research, meaning it's not really 'ghetto' surgery and arguably harder to get than the professional augments if someone takes one of the two pairs of goggles on the station.

**Comment by paprka** [comment]
I could have swore someone put welding goggles in engineering. Those need to be more prolific, honestly.

Flashlight augs being ghetto surgery'd isn't really of consequence to me because ghetto flashlight aug is basically a flashlight cavity implant anyway.

**Comment by paprka** [comment]
@Stickymayhem You need to open the panel with a screwdriver, but yes, you can pulse wires without having a mulitool in hand.

**Comment by Razharas** [comment]
Meh all have same tech levels

File: code/modules/surgery/misc_augments.dm [hunk 1]
```diff
@@ -0,0 +1,120 @@
+/* HACKING AUGMENT */
+// Allows the user to pulse wires in hack panels without a mulitool.
+
+/datum/surgery/hacking_augment
+	name = "hacking augment"
+	steps = list(/datum/surgery_step/incise, /datum/surgery_step/retract_skin, /datum/surgery_step/clamp_bleeders, /datum/surgery_step/insert_hack, /datum/surgery_step/close)
+	species = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
+	location = "head"
```
Comment (Razharas) [inline code/modules/surgery/misc_augments.dm]: Well at least not groin

**Comment by Razharas** [comment]
Also i dont like the substitutes as steps of operation instead of actual implants
Its like if you will try to apply a mop 20 times instead of borg leg you will eventually get a proper augmented leg out of mop

**Comment by Razharas** [comment]
Also without quadratic lights implant with PDA active makes you very bright

**Comment by Tenebrosity** [comment]
These just seem like a hassle for such minor bonuses. I'd rather just keep my multitool in my belt.

**Comment by Jordie0608** [comment]
Changelog.

**Comment by paprka** [comment]
Gonna rework this I think

**Pull request closed without merging**